apps/api/src/notifications/services/notification/notification.service.ts

```diff
@@ -7,7 +7,7 @@ const DEFAULT_BACKOFF_OPTIONS: BackoffOptions = {
   maxDelay: 5_000,
   startingDelay: 500,
   timeMultiple: 2,
-  numOfAttempts: 10
+  numOfAttempts: 5
```

Comment:
reduced it because in case of failure, login request can stuck for about 30+ seconds